src/openrct2/peep/peep.c

```diff
@@ -1392,12 +1392,14 @@ static uint8 peep_get_action_sprite_type(rct_peep* peep)
 	}
 }
 
-void sub_693B58(rct_peep* peep){
+void peep_update_current_action_sprite_type(rct_peep* peep){
```

Comment:
Hm, are we sure it's not worth keeping these easy notes in case we messed up decompilation? I don't have a strong opinion on this in any case.